web-wallet/vite-setup.js

```diff
@@ -33,6 +33,7 @@ vi.spyOn(global, "btoa").mockImplementation(data => Buffer.from(data, "binary").
 // Adding missing bits in JSDOM
 
 vi.mock("./src/lib/dusk/mocks/IntersectionObserver");
+vi.mock("./src/lib/dusk/mocks/ResizeObserver");
```

Comment:
Seems like this fixes the bug, however, the tests are failing for the Tabs component.

<img width="1480" alt="Screenshot 2024-02-08 at 15 07 20" src="https://github.com/dusk-network/rusk/assets/34094428/a58fddf6-be7f-42f1-9ae9-ecdf1e4dc430">

(and the snapshot changed).

Wondering if the mock has to be tweaked too or is it some other config? cc: @ascartabelli 🙈